pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -15,8 +21,22 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
+  postPatch = ''
+    sed -i 's# /bin/echo# echo#' Makefile
+  '';
+
   preBuild = ''
-    make cli-deps-static-sites
+    ln -s ${nodeDependencies}/lib/node_modules web/node_modules
+    DISABLE_ESLINT_PLUGIN=true make web-build docs-build
```

Comment:
Can you set `npm_config_offline=true` here too as well? This fixes the [failure on Darwin](https://logs.nix.ci/?key=nixos/nixpkgs.194238&attempt_id=476b5968-8c55-4dba-8d4f-64ddd0d94b5a).

Reply:
Added. I don't have darwin myself, so I can't test locally. Let's see if ofborg thinks it's okay now.

Reply:
It did 👍